src/ApiDataCache.js

```diff
@@ -4,30 +4,32 @@ import mkdirp from "mkdirp";
 import ConfigurationError from "./ConfigurationError";
 
 export default class ApiDataCache {
-  constructor(host, {rootPath, cacheDir}) {
+  constructor(host, { rootPath, cacheDir }) {
     this.host = host;
     const dir = this.dir = cacheDir && path.join(rootPath, cacheDir, host);
 
     if (dir) {
       try {
         mkdirp.sync(dir);
       } catch (e) {
-        throw new ConfigurationError(`Can't use cacheDir "${cacheDir}" (${e.message})`);
+        throw new ConfigurationError(
+          `Can't use cacheDir "${cacheDir}" (${e.message})`
+        );
       }
     }
   }
 
   get(type, key) {
-    if (!this.dir) return;
+    if (!this.dir)
+      return;
     try {
       return fs.readFileSync(this.fn(type, key), "utf-8");
-    } catch (e) {
-      // Pass.
-    }
+    } catch (e) {}
```

Comment:
Not really, I just used auto-formatting from [prettier](https://github.com/jlongster/prettier).
Happy to revert it if necessary.